app/experimenter/base/management/commands/load_dummy_experiments.py

```diff
@@ -23,8 +23,8 @@ def handle(self, *args, **options):
             logger.info("Created {}: {}".format(experiment, status))
 
         for application in NimbusExperiment.Application:
-            for status in NimbusExperiment.Status:
-                experiment = NimbusExperimentFactory.create_with_status(
-                    status, application=application
+            for lifecycle in NimbusExperiment.Lifecycles:
+                experiment = NimbusExperimentFactory.create_with_lifecycle(
+                    lifecycle, application=application
                 )
```

Comment:
Also, trying to decide if this is a good idea. It generates a *lot* of dummy experiments, but it does result in examples at every stage of launch & end flows